### PR TITLE
Class 05 Homework Complete

### PR DESCRIPTION
**Pull request by blackeaglejs**
comfort level: 4

### CODE REVIEW

**Comment by blackeaglejs** [comment]
@mobjohnson Please see the homework here.

File: class04/zoheb/FortuneCookie.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+require 'pry'
```
Comment (mobjohnson) [inline class04/zoheb/FortuneCookie.rb]: This file name should be snake case, fortune_cookie.rb

File: class04/zoheb/FortuneCookie.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		if @wrapped == true
+			return "YOu need to unwrap it, then crack it."
+		elsif @wrapped == false && @cracked == false
+			return "You still need to crack it."
+		elsif @wrapped == false && @cracked == true
+			return @fortune.sample
+		end
+	end
+end
+
```
Comment (mobjohnson) [inline class04/zoheb/FortuneCookie.rb]: looks good!

File: class04/zoheb/scrabble_word.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      @total += @points[letter]
+    end
+    return @total
+  end
+
+  def multiplier_score(mult)
+    return @total*mult
+  end
+end
+
```
Comment (mobjohnson) [inline class04/zoheb/scrabble_word.rb]: Nicely done!

**Comment by mobjohnson** [comment]
I have reviewed your homework and show you as completed for this homework assignment.

**Pull request merged**